spot_wrapper/wrapper.py

```diff
@@ -1949,6 +1946,18 @@ def grasp_3d(self, frame, object_rt_frame):
 
     ###################################################################
 
+    def _init_current_graph_nav_state(self):
+        # Store the most recent knowledge of the state of the robot based on rpc calls.
+        self._current_graph = None
+        self._current_edges = dict()  # maps to_waypoint to list(from_waypoint)
+        self._current_waypoint_snapshots = dict()  # maps id to waypoint snapshot
+        self._current_edge_snapshots = dict()  # maps id to edge snapshot
+        self._current_annotation_name_to_wp_id = dict()
+        self._current_anchored_world_objects = (
+            dict()
+        )  # maps object id to a (wo, waypoint, fiducial)
+        self._current_anchors = dict()  # maps anchor id to anchor
```

Comment:
I like the convention `_current_anchors_by_id`, which IMO removes the need for a comment.`. I suppose more explicit would be `_current_anchors_by_anchor_id`

Reply:
Good point. I think in the context of spot_ros2, just looking at `id` one wouldn't know which id it is...